### ISSUE TITLE
Publish tmLanguage as a Sublime Text package?

### ISSUE THREAD

**Issue by domenic** [speced/bikeshed#214]
I can understand if you don't want to ("support my package manager!" bugs suck), but it'd make things easier for some of us.

**Comment by nox** [comment]
Any news on this?

**Comment by tabatkins** [comment]
Nope. Cleanup is even *more* necessary now, as I haven't updated the file in a long time, and I still need to actually figure out how to publish it.

**Comment by kvark** [comment]
@tabatkins the way I set this up is by "Add Repository" pointing to your github. This way I'm getting any upstream fixes/followups to the syntax file. I think it works similarly for "publishing" the package - it still gets tied to your github, and you keep working on improving the syntax There is no downside of registering the package today.

**Comment by tabatkins** [comment]
Oh dang, so you're saying i can just register my repo in Sublime's package manager and it works without extra effort on my part?

**Comment by kvark** [comment]
Yeah, see https://packagecontrol.io/docs/submitting_a_package#Step_4
Registering just requires you to point to this github (although, it might make sense to use a separate repo). Versioning is with tags.

**Comment by domenic** [comment]
I no longer need Sublime Text support, and @alexnj's [visual-bikeshed](https://github.com/alexnj/visual-bikeshed) has good syntax highlighting, so I'll close this.
